client/client-multi/client-c/src/kaa/kaa.c

```diff
@@ -118,12 +121,15 @@ static kaa_error_t kaa_context_create(kaa_context_t **context_p, kaa_logger_t *l
 
     if (!error)
         error = kaa_bootstrap_manager_create(&((*context_p)->bootstrap_manager), (*context_p)->channel_manager,
-                                             (*context_p)->logger);
+                                             (*context_p)->logger, (*context_p));
```

Comment:
I think we can use only two parameters - &((*context_p)->bootstrap_manager), *context_p